footer.php

```diff
@@ -7,15 +7,21 @@
 				<a href="https://creativecommons.org" class="main-logo margin-bottom-bigger">
 					<span class="has-text-white">
 						<svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidyMid meet" viewBox="0 0 304 73">
-							<use  xlink:href="<?php echo get_bloginfo( 'template_directory' ) . '/assets/img/logos/cc/logomark.svg#creativecommons'; ?>"></use>
+							<use  xlink:href="<?php echo get_bloginfo( 'template_directory' ) . '/assets/img/logos/cc/logomark.svg#logomark'; ?>"></use>
 						</svg>
 					</span>
 				</a>
 				<p>
 					<address class="margin-bottom-normal"> Creative Commons <br> PO Box 1866, Mountain View CA 94042</address>
-					<a href="mailto:info@creativecommons.org" class="mail">info@creativecommons.org</a>
+					<a href="mailto:info@creativecommons.org" class="mail">info@creativecommons.org</a><br />
 					<a href="tel://+1-415-429-6753" class="phone">+1-415-429-6753</a>
 				</p>
+				<div class="social-media">
+					<a href="#" class="social instagram"><i class="icon instagram"></i></a>
```

Comment:
I think it's missing the social media links here
You can copy it from [here](https://github.com/creativecommons/cccatalog-frontend/blob/develop/src/components/FooterSection.vue#L23)

Reply:
Done. I added the social media links.
what do you say about icons?

Reply:
I mean the CC logos and license icons